bio/reference/ensembl-genomes-sequence/wrapper.py

```
@@ -0,0 +1,116 @@
+# TODO: bacteria, fungi/ascomycota folders are further subdivided
+# TODO: chromosome file names may not follow standard naming convention, e.g.
+# Plasmodium_falciparum.GCA000002765v3.dna.primary_assembly.Pf3D7_01_v3.fa.gz
+
+if path.splitext(snakemake.output[0])[1] == ".gz":
+    decompress = ""
+else:
+    decompress = " | gzip -d"
+
+suffixes = []
+datatype = snakemake.params.get("datatype", "")
+chromosome = snakemake.params.get("chromosome", "")
```

Ruff (0.14.10)

Undefined name `snakemake`

(F821)